lib/kiev/context_reader.rb

```diff
@@ -6,23 +6,28 @@ module Kiev
   # change field lookup.
   class ContextReader
     REQUEST_ID = "request_id"
+    TRACKING_ID = "tracking_id"
     REQUEST_DEPTH = "request_depth"
     TREE_PATH = "tree_path"
 
     def initialize(subject)
+      subject[TRACKING_ID] ||= subject[REQUEST_ID]
```

Comment:
Why are you setting Request ID to Tracking ID?

Reply:
to have consistent and same value in both fields always. With priority to tracking_id.